hypervisor/src/arch/x86/emulator/instructions/mod.rs

```diff
@@ -124,7 +124,7 @@ fn memory_operand_address<T: CpuStateManager>(
         address += index;
     }
 
-    address += insn.memory_displacement64();
+    address = address.wrapping_add(insn.memory_displacement64());
```

Comment:
I would prefer the first line of the commit message to be `hypervisor: emulator: Use wrapping add for memory offset`

Reply:
fixed